index.html

```diff
@@ -1 +1,132 @@
-Hello word
\ No newline at end of file
+<html>
+
+<head>
+    <title>first page</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+    <header>
+        <div>
+            <div>
+                <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+                <span> Nifty Penguin Magic </span>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="#"> npm Enterprise </a></li>
+                    <li><a href="#"> Products </a></li>
+                    <li><a href="#"> Solutions </a></li>
+                    <li><a href="#"> Resources </a></li>
+                    <li><a href="#"> Docs </a></li>
+                    <li><a href="#"> Support </a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div>
+            <img class="logo" src="images/npm-logo.png" alt="logo" />
+            <img class="search" src="images/magnifying-glass.png" alt="search" />
+            <input type="text" name="searchtext" id="searchtext" />
+            <button class="red-button" name="Search" id="search">Search</button>
+            <button>Join</button>
+            <button>Log in</button>
+        </div>
+    </header>
+    <section>
+        <div class="back">
+            <div class="content1">
+                <heading>
+                    <h1>Build Amazing Things</h1>
+                </heading>
+                <article>
+                    <h2>Essential JavaScript development tools that help you go to market faster and build powerful
+                        applications using
+                        modern open source code.</h2>
+                </article>
+                <div>
+                    <a href="#"><button class="red-button">
+                            See plans
+                        </button></a>
+                    <a href="#"><button>
```

Comment:
Ojo aquí crea las clases que necesites para agregar estilos a tus botones al igual del Log in e Join